sphinx/developers/java-library.rst

```diff
@@ -188,32 +188,32 @@ The complete list of third-party dependencies for `formats-gpl.jar` is as follow
     * - `j2objc-annotations v1.3 <https://developers.google.com/j2objc/>`_
       - com.google.j2objc:j2objc-annotations:1.3
       - `Apache License v2.0`_
-    * - `httpservices v5.3.3 <https://www.unidata.ucar.edu/software/netcdf-java/v4.6/reference/httpservices.html>`_
-      - edu.ucar:httpservices:5.3.3
-      - `MIT-Style License`_
-    * - `httpclient v4.5.9 <https://hc.apache.org/>`_
-      - org.apache.httpcomponents:httpclient:4.5.9
+    * - `httpservices v5.6.0 <https://www.unidata.ucar.edu/software/netcdf-java/v4.6/reference/httpservices.html>`_
```

Comment:
Update URL to https://docs.unidata.ucar.edu/netcdf-java/5.6/userguide/index.html or similar, since the current one references 4.6?

Reply:
Done. For both links I chose to point to the canonical entry point of the overarching NetCDF-Java library,